tests/components/lock/test_init.py

```diff
@@ -131,15 +132,15 @@ async def test_lock_open_with_code(
     state = hass.states.get(mock_lock_entity.entity_id)
     assert state.attributes["code_format"] == r"^\d{4}$"
 
-    with pytest.raises(ValueError):
+    with pytest.raises(ServiceValidationError):
```

Comment:
Should we check the message using `match`?

Reply:
Put in the check in the last raise